addons/sale/models/sale.py

```diff
@@ -1118,7 +1118,8 @@ def _update_line_quantity(self, values):
         orders = self.mapped('order_id')
         for order in orders:
             order_lines = self.filtered(lambda x: x.order_id == order)
-            msg = "<b>The ordered quantity has been updated.</b><ul>"
```

Comment:
If you want to avoid html tags in the translation, let's push it to the max:
```py
msg = "<b>" + _("The ordered quantity has been updated.") + "</b></ul>"
```